docs/FEDERATION-DOWNLOAD.md

```diff
@@ -0,0 +1,92 @@
+
+On the download of keys from the federation gateway a process of normalization needs to take place. This is done to enable the keys to be consumable by the DE CWA app as not all countries support the same approach which is required for the CWA app. The means the following:
+
+- For keys from countries which support the daysSinceOnset scenario will need to be converted into an appropriate transmission risk level
+- For keys from countries which support the transmission risk level we might need to translate the value to something which is reasonable for the RKI
+- For keys from countries which support neither, or a key doesn't happen to have either value then there will be a need to provide a reasonable default
+
+The rules above would be defined at 2 levels:
+
+- On a per country basis such that the rules can be specified and values derived based on the origin country
+- At a global level for situations where we do not have country specific rules
+
+They would also take into consideration other attributes provided within the key data for example report type. The full set of attributes to be evaluated and how is still TBD.
+
+These rules will allow the keys sourced from the federation gateway to be processed within the CWA App and be considered with the risk detection algorithms.
+
+## External Dependencies
+
+- **Vault**: Used for secrets and certificate storage
+- **RDBMS**: PostgreSQL as the persistent storage for keys which are downloaded
+- **Federation Gateway Service**: The service where the service downloads the keys
+
+## Spring Profiles
+
+Spring profiles are used to apply federation key download service configuration based on the running environment, determined by the active profile.
+
+You will find `.yaml` and `.xml` based profile-specific configuration files at [`/services/federation-download/src/main/resources`](/services/federation-download/src/main/resources).
+
+### Available Profiles
+
+Profile                                           | Effect
+--------------------------------------------------|-------------
+`dev`                                             | Sets the log level to `DEBUG` and changes the `CONSOLE_LOG_PATTERN` used by Log4j 2.
+`cloud`                                           | Removes default values for the `spring.flyway`, `spring.datasource` and sets federation gateway contexts
+`disable-ssl-server`                              | Disables SSL for the submission endpoint.
```

Comment:
There is not 'disable-ssl-server' for the download service

Reply:
removed